db/migrate/20210114074037_add_subscription_tags.rb

```diff
@@ -0,0 +1,32 @@
+#  Copyright (c) 2012-2020, CVP Schweiz. This file is part of
+#  hitobito_cvp and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito.
+
+class AddSubscriptionTags < ActiveRecord::Migration[6.0]
+  def up
+    create_table :subscription_tags do |t|
+      t.boolean :excluded, default: false
+
+      t.references :subscription, foreign_key: true, type: :integer, null: false
+      t.references :tag, foreign_key: true, type: :integer, null: false
+    end
+
+    Subscription.groups.find_each do |subscription|
+      tagging = ActsAsTaggableOn::Tagging.find_by(taggable_id: subscription.id,
+                                                   taggable_type: Subscription.sti_name)
```

Comment:
shouldn't we also remove existing tagging entry after migrating ?

Reply:
maybe it's a good idea to write a spec for this migration. 
example: https://github.com/hitobito/hitobito/blob/messages/spec/migrations/migrate_mail_log_spec.rb